hack/lib/constants.sh

```diff
@@ -335,7 +335,7 @@ function os::build::check_binaries() {
   # enforce that certain binaries don't accidentally grow too large
   # IMPORTANT: contact Clayton or another master team member before altering this code
   if [[ -f "${OS_OUTPUT_BINPATH}/${platform}/oc" ]]; then
-    if [[ "$(du -m "${OS_OUTPUT_BINPATH}/${platform}/oc" | cut -f 1)" -gt "110" ]]; then
+    if [[ "$(du -m "${OS_OUTPUT_BINPATH}/${platform}/oc" | cut -f 1)" -gt "115" ]]; then
```

Comment:
@smarterclayton I clearly don't understand how the binary gets reflected. This doesn't look like it adds a net new dependency to `oc`.

Reply:
@smarterclayton wouldn't it make more sense to have this as ./hack/verify-binary-size.sh or something? no idea what the binary size is on windows and this will probably error out when building on windows anyway as you missing the `.exe` suffix ;-)

Reply:
The point is to catch people doing stupid things.  It’s possible this is bumping because 1.9.4 increased size of binary over 1.9.1.  115 is fine for now, I need to look.

We don’t build in ci on Windows so it’s not a problem I’m worried about.